content/applications/inventory_and_mrp/inventory/barcode/operations/receipts_deliveries.rst

```diff
@@ -0,0 +1,183 @@
+
+Scan barcodes for delivery orders
+=================================
+
+To process warehouse deliveries for outgoing products, there first needs to be a sales order (SO)
+created, and a delivery operation to process.
+
+To create a :abbr:`SO (sales order)`, navigate to the :menuselection:`Sales app --> Create` to
+create a new quotation. From the blank quotation form, click the drop-down menu next to the
+:guilabel:`Customer` field to add a customer. Then, on the :guilabel:`Product` line under the
+:guilabel:`Order Lines` tab, click :guilabel:`Add a product`, and select the desired product(s) to
+add to the quotation.
+
+Once ready, click :guilabel:`Save` at the top of the form, and click :guilabel:`Confirm Order` to
+confirm the quotation to a :abbr:`SO (sales order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-sales-order.png
+   :align: center
+   :alt: Completed sales order for barcode product.
+
+To process and scan barcodes for warehouse deliveries, navigate to the :menuselection:`Barcode app`.
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To process deliveries, click on the :guilabel:`Operations` button at the
+bottom of the screen. This navigates to an :menuselection:`Operations` overview page.
+
+From this page, locate the :guilabel:`Delivery Orders` card, and click the :guilabel:`# To Process`
+button to view all outstanding deliveries. Then, select the desired delivery order to process. This
+navigates to the barcode transfer screen.
+
+.. image:: receipts_deliveries/receipts-deliveries-operations-page.png
+   :align: center
+   :alt: Operations overview page in Barcode app dashboard.
+
+From this screen, an overview of all deliveries to process within that transfer (**WH/OUT/000XX**)
+is shown. At the bottom of the screen, there are options to :guilabel:`Add Product` or
+:guilabel:`Validate`, depending on if products need to be added to the operation, or if the whole
+operation should be validated at once.
+
+To process and scan each product individually, choose a specific product line. The :guilabel:`+1`
+button can be clicked to indicate delivery of that product, or the :guilabel:`pencil icon` can be
+clicked to open a new screen to edit that product line.
+
+From this screen, the product that's being delivered is listed. Under the product name, the
+:guilabel:`Quantity` line can be edited. Either change the `0` in the line to the desired quantity,
+or click the :guilabel:`/# Units` button (in this case, :guilabel:`/10 Units`) to automatically fill
+the quantity ordered from the :abbr:`SO (sales order)`.
+
+Additionally, the :guilabel:`+1` and :guilabel:`-1` buttons can be clicked to add or subtract
+quantity of the product, and the :guilabel:`number keys` can be used to add quantity, as well.
+
+Below the :guilabel:`number keys` is the :guilabel:`location` line, which reads `WH/Stock` by
```

Comment:
Since the words "number keys" and "location" aren't explicitly shown on the UI, according to our documentation guidelines, we shouldn't use guilabel in this case. I think italics  would be the move here (: